src/view/services/command.manager.js

```diff
@@ -0,0 +1,12 @@
+export class CommandManager {
+	constructor(apply) {
+		this.apply = apply;
+	}
+
+	execute(commands) {
+		// First we need to get list of executable commands, cause execution of prev command can
```

Comment:
What do you mean by sequentially?
Real example for enter shortcut:
```click enter: open popup -> set popup open = true```
```click enter: if popup opened close it```
So popup for the user won't open.
We just need to manipulate with command sets, but this is kind of big refactoring